### PR TITLE
Update to latest internal version

### PR DESCRIPTION
**Pull request by rok-povsic**

### CODE REVIEW

File: sonar-project.properties [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# sonarqube server url
+sonar.host.url=https://sonarqube.cf.ai
```
Comment (zeiler) [inline sonar-project.properties]: @robertwenquan please review to make sure there are no secrets in this PR please. This is merging all private repo into public for good.

File: scripts/sync_repo.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
```
Comment (zeiler) [inline scripts/sync_repo.sh]: Do we need this file?

Comment (rok-povsic) [reply scripts/sync_repo.sh]: No, good catch. Removed.

File: sonar-project.properties (hunk 1 above)
Comment (robertatclarifai) [reply sonar-project.properties]: there is no secrets in this file. But this file might not be good for public consumption. 
The `sonarqube.cf.ai` itself should be internal only

File: tests/requirements.txt [hunk 3]
```diff
@@ -10,4 +10,4 @@ pytest-cov==2.6.0
 pytest-xdist==1.23.2
 responses>=0.5
 retrying==1.3.3
-yapf==0.22.0
+yapf==0.19.0
```
Comment (robertatclarifai) [inline tests/requirements.txt]: why this is downgraded?

Comment (zeiler) [reply tests/requirements.txt]: To match main repo

**Pull request merged**